concordium-sdk/src/main/java/com/concordium/sdk/responses/poolstatus/PendingChangeReduceBakerCapital.java

```diff
@@ -0,0 +1,24 @@
+package com.concordium.sdk.responses.poolstatus;
+
+import com.concordium.sdk.transactions.CCDAmount;
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.Date;
+
+@Getter
+@ToString
+@Data
+public class PendingChangeReduceBakerCapital extends PendingChange {
```

Comment:
Make fields final please.
I don't see the need for @Getter @ToString when @Data is used.

Reply:
done